bundle.Dockerfile

```diff
@@ -5,8 +5,8 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=nvidia-network-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=v24.7.0-rc.1,stable
-LABEL operators.operatorframework.io.bundle.channel.default.v1=v24.7.0-rc.1
+LABEL operators.operatorframework.io.bundle.channels.v1=v24.7.0,stable
```

Comment:
for 24.10 will we have "v24.10,stable" as channel.v1 value ?

Reply:
Yes.
Should be handle in workflow:
https://github.com/Mellanox/network-operator/pull/1034/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR131

Reply:
Do you think we should have here "v24.7.1", like we did in "v24.4.1"?
https://github.com/redhat-openshift-ecosystem/certified-operators/blob/main/operators/nvidia-network-operator/24.4.1/metadata/annotations.yaml#L8